Library/Homebrew/test/test_language_python.rb

```diff
@@ -71,18 +73,24 @@ def test_pip_install_accepts_resource
     @venv.pip_install res
   end
 
-  def test_link_scripts_links_scripts
-    bin = (@dir/"bin")
-    dest = (@dir/"dest")
-    bin.mkpath
-    refute((bin/"kilroy").exist?)
-    refute((dest/"kilroy").exist?)
+  def test_pip_install_and_link_links_scripts
+    bin = (@dir/"bin").tap(&:mkpath)
+    dest = @formula.dest
```

Comment:
Indeed. That's a stupid last-minute change I made because I was shuffling variable names back and forth and sadly I failed to run the test suite afterwards.